src/App.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
-import './App.scss';
+import classNames from 'classnames/bind';
```

Comment:
```suggestion
import classNames from 'classnames';
```
Этого достаточно, никакого bind не надо